SetMultiScreensVideoMode.brs

```diff
@@ -1,41 +1,57 @@
 ' Multi Screens VideoMode Plugin Example
 ' Plugin function name - setvideomode
-' Plugin return type - As Array: see below example, it is plugin's responsibility to 
+' Plugin return type - As Object (array): see below example, it is plugin's responsibility to 
 ' return the same format as GetScreenModes() returns. Then the presentation autorun will
 ' pick up your ScreenConfigList and set video modes for your screens.
 
-Function setvideomode(videoModeInputs, bsp)
+' Note: This plugin assumes it is executed on BrightSign Model with Multiple HDMI outputs.
+
+' Minimum BrightAuthor:connected version: v1.11.0
+
+Function setvideomode(videoModeInputs, bsp) As Object

```

Comment:
Can we update function name to pascal case?

Reply:
It was updated to camel case instead of pascal. Personally I use pascal for BrightScript function name. Not a big deal to use camel.

Reply:
Pascal is fine. Updated.